RevenueCatUI/Templates/V2/Components/Carousel/CarouselComponentView.swift

```diff
@@ -131,13 +133,19 @@ private struct CarouselView<Content: View>: View {
 
     /// A timer for auto-play, if enabled.
     @State private var autoTimer: Timer?
+
     @State private var isPaused: Bool = false
     @State private var pauseEndDate: Date?
+    @State private var isInitialized = false
+
+    /// Used to keep the drag position for better animations
+    @State private var dragOffset: CGFloat = 0
```

Comment:
After finger lifts of page after dragging, it would instantly pop back without this